libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -568,9 +569,11 @@ - (void)beginUserAgentFlow {
                 json = [NSJSONSerialization JSONObjectWithData:data options:0 error:&jsonError];
                 if (nil == jsonError && [json isKindOfClass:[NSDictionary class]]) {
                     NSDictionary *dict = (NSDictionary *)json;
-                    if (dict[kSFOAuthAccessToken]) {
+                    [self updateCredentials:dict];
```

Comment:
The data that comes back from the swap call is the data that bootstraps the rest of the process.